src/options.js

```diff
@@ -11,8 +13,16 @@ var defaults = {
   bracketSpacing: true
 };
 
+var exampleConfig = Object.assign({}, defaults, {
+  filename: "testFilename",
+  useFlowParser: false,
+  printWidth: 80,
+  originalText: "text"
+});
+
 // Copy options and fill in default values.
 exports.normalize = function(options) {
+  validate(options, { exampleConfig });
```

Comment:
`originalText` isn't really an option. It's a hack to allow the internal parser to get access to it, but it shouldn't be considered an option.